nilearn/decoding/decoder.py

```diff
@@ -637,7 +637,7 @@ def fit(self, X, y, groups=None):
 
         """
         check_params(self.__dict__)
-        self.estimator = _check_estimator(self.estimator)
+        self.estimator_ = _check_estimator(self.estimator)
```

Comment:
Could we do this in `__init__`?

Reply:
nope: by sklearn convention attributes with trailing '_' are created at fit time